packages/aquarius/src/bot/plugins/nice.js

```diff
@@ -16,7 +16,7 @@ export const info = {
 export default async ({ aquarius, analytics }) => {
   aquarius.onMessage(info, (message) => {
     if (
-      message.cleanContent.includes('69') ||
```

Comment:
How is this better than `includes()`?

Reply:
It will not react to situations when "69" is found in the snowflake.